src/lando/ui/legacy/forms.py

```diff
@@ -85,8 +84,10 @@ class UpliftRequestForm(UpliftAssessmentForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        uplift_repos = get_uplift_repositories()
-        self.fields["repository"].choices = [(repo, repo) for repo in uplift_repos]
+        uplift_repos = Repo.objects.filter(approval_required=True).all()
+        self.fields["repository"].choices = [
+            (repo.name, repo.name) for repo in uplift_repos
```

Comment:
nit: Maybe we should have a better display name, since we could probably parse the name and branch now from the repo details.

Reply:
Agreed, it would be nice to have a readable name field on the `Repo` class. For now I think staying with the same format is acceptable.